tests/page-objects/apdex/page.js

```diff
@@ -160,45 +185,51 @@ module.exports = class Page {
     await this.navigate(form.postTestPath);
   }
 
-  async relaunchApp(settingsProvider) {
-    const commonElements = settingsProvider.getCommonElements();
+  async relaunchApp(commonElements) {
     const UI_ELEMENT = commonElements?.relaunchAppAssert || '//*[@text="People"]';
     await driver.execute('mobile: terminateApp', {appId: 'org.medicmobile.webapp.mobile'});
     await driver.execute('mobile: activateApp', {appId: 'org.medicmobile.webapp.mobile'});
     await this.waitForDisplayedAndRetry(UI_ELEMENT);
   }
 
-  async searchContact(form) {
-    const page = form.pages[0];
+  async search (page, commonElements) {
+    if (!page || !page.search) {
+      return;
+    }
 
-    await this.navigate(form.navigation);
-    await this.fillUpFormPage(page);
-    await this.assertMany(form.postSubmitAsserts);
+    const SEARCH_ICON = commonElements?.searchIcon || '//android.widget.TextView[@text=""]';
+    const SEARCH_INPUT = '//android.widget.EditText';
```

Comment:
@ralfudx @tatilepizs, I need help with this one. I didn't find how to improve these selectors.
Can you please help me?

Reply:
Thanks for checking @ralfudx. I agree that it is better not to use the index for finding this element